marshalers.go

```diff
@@ -9,16 +9,15 @@ import (
 	"unsafe"
 
 	"github.com/cilium/ebpf/internal"
-
-	"github.com/pkg/errors"
+	"golang.org/x/xerrors"
```

Comment:
Please keep the empty line if it was already there. I've picked up the convention to split imports into stdlib, same package, dependencies blocks.
```suggestion

	"golang.org/x/xerrors"
```